data/service/context/standard.go

```diff
@@ -25,9 +25,9 @@ type Standard struct {
 	permissionClient        permission.Client
 	dataDeduplicatorFactory deduplicator.Factory
 	dataStoreDEPRECATED     dataStoreDEPRECATED.Store
-	dataSession             dataStoreDEPRECATED.DataSession
+	dataCollection          dataStoreDEPRECATED.DataRepository
```

Comment:
I think naming should be improved to be more consistent - we should stick with either one of `repository`, `collection` or `session`. What do you think?

Reply:
Yes, we should. I had tried to standardise on `repository`, but you found one that slipped through. Thanks!